src/tests/api.rs

```diff
@@ -52,7 +52,8 @@ fn exercise_full_api() {
         parse: ParseOptions {
             smart: false,
             default_info_string: Some("abc".to_string()),
-            relaxed_tasklist_matching: true,
+            relaxed_tasklist_matching: false,
```

Comment:
I updated this because the other options/extensions are turned off, including the task list extension.  So it seems like that's the correct state to start off with.

Reply:
Any value is fine; we don't check the output! The purpose of this file was to notice when the API changed shape in any way, so I'd notice before I made a semver-breaking release.

I'll rewrite these to use the builders in a moment; there's no point to this test as written, since non-exhaustive means the structs can't be made this way from outside the crate.